lib/idx/remediators/Base/VerifyAuthenticator.ts

```diff
@@ -13,14 +13,21 @@
 
 import { Remediator, RemediationValues } from './Remediator';
 import { getAuthenticator, Authenticator } from '../../authenticator';
-import { IdxRemediation } from '../../types/idx-js';
+import { IdxRemediation, IdxContext } from '../../types/idx-js';
+import { NextStep } from '../../types';
 
 export interface VerifyAuthenticatorValues extends RemediationValues {
```

Comment:
Maybe `VerifyAuthenticatorValues` should aggregate values from different Authenticators? We have this pattern a few other places in `auth-js`

reference: https://github.com/okta/okta-auth-js/blob/master/lib/idx/proceed.ts#L27